doc/classes/RenderSceneBuffers.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="RenderSceneBuffers" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A scene buffers object is created for each viewport for which 3D rendering is done.
+	</brief_description>
+	<description>
+		A scene buffers object is created for each viewport for which 3D rendering is done. It manages any additional buffers used during rendering and will discard buffers when the viewport is resized.
+		[b]Note:[/b] this is an internal rendering server object only exposed for GDExtension plugins.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="configure">
+			<return type="void" />
+			<param index="0" name="render_target" type="RID" />
```

Comment:
I am a bit worried about this because it means any change here will break compatibility. Maybe we should have an object RenderSceneBuffersConfiguration we can pass, so we can keep adding things to it if we need and avoid breaking compat?

To clarify this is an issue with virtual methods, which are not as easy to keep from compat breakage as regular methods.